javaslang-gwt/src/test/java/client/CollectionsTestGwt.java

```diff
@@ -60,7 +60,8 @@ public void testCompileTreeSet() {
         applyCollection(chars -> TreeSet.ofAll(Iterator.ofAll(chars)));
     }
 
+    @SuppressWarnings("Convert2MethodRef")
     public void testCompileVector() {
-        applyCollection(Vector::ofAll);
+        applyCollection(chars -> Vector.ofAll(chars));
```

Comment:
At least there are chances that it will be fixed near-time. Our GWT dependency is still an RC (release candidate).